lms/envs/common.py

```diff
@@ -1838,6 +1838,7 @@
 DEFAULT_PRIORITY_QUEUE = 'edx.core.default'
 LOW_PRIORITY_QUEUE = 'edx.core.low'
 HIGH_MEM_QUEUE = 'edx.core.high_mem'
+GRADES_BACKFILL_QUEUE = 'edx.core.grades_backfill'
```

Comment:
I'm not seeing where the task specifies to use this newly defined queue?

Reply:
That would be on the command line here:

https://build.stage.lagunita.stanford.edu/view/grades/job/backfill_persisent_grades/configure

Reply:
@dcadams Can you commit that script to the `configuration` repo, along with the other jenkins jobs?

`configuration:stanford/bin`